active_model_serializers.gemspec

```diff
@@ -59,4 +59,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'grape', ['>= 0.13', '< 1.0']
   spec.add_development_dependency 'json_schema'
   spec.add_development_dependency 'rake', ['>= 10.0', '< 12.0']
+  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'm'
```

Comment:
I don't like putting anything in the gemspec not required for running the tests.  I'd rather this be proposed to the Gemfile in a separate PR.  Since this has come up a couple of times and I basically have the same thing in my `Gemfile.local` I think it's worth adding